web-ng/src/app/shared/converters/firebase-data-converter.ts

```diff
@@ -366,9 +366,10 @@ export class FirebaseDataConverter {
    * @param id the uuid of the project instance.
    * @param data the source data in a dictionary keyed by string.
    */
-  static toFeature(id: string, data: DocumentData): Feature | undefined {
+  static toFeature(id: string, data: DocumentData): Feature | void {
     if (!this.isFeatureValid(data)) {
-      console.warn(`Invalid feature ${id} in remote data store ignored`);
+      const errors = this.getFeatureErrors(id, data);
```

Comment:
Why split error state out from conversion logic? I'd think we'd do them at the same time so that we don't need to manage drift between conversion logic and `getFeatureErrors`.